<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="kiGuess">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>I think it is: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="team1Text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Team 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="team1Score">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="team2Score">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="team2Text">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Team 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>150</y>
     <width>673</width>
     <height>511</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>673</width>
     <height>511</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <zorder>startScreen</zorder>
   <zorder>startScreen</zorder>
   <zorder>startScreen</zorder>
  </widget>
  <widget class="QPushButton" name="startGame">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>330</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Round</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>390</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Image</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timer">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Awesome Wii Draw Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QSlider" name="secondsSlider">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="selectSeconds">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="value" stdset="0">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="secondText">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose Seconds for each Round</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endGame">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>450</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>End Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="startScreen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-6</y>
     <width>1011</width>
     <height>711</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/table.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="redTeam">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/teamDots/redDot.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="blueTeam">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/teamDots/blueDot.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Term:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="category">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>250</y>
     <width>67</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>kiGuess</zorder>
  <zorder>team1Text</zorder>
  <zorder>team1Score</zorder>
  <zorder>team2Score</zorder>
  <zorder>team2Text</zorder>
  <zorder>frame</zorder>
  <zorder>startGame</zorder>
  <zorder>clear</zorder>
  <zorder>timer</zorder>
  <zorder>endGame</zorder>
  <zorder>redTeam</zorder>
  <zorder>blueTeam</zorder>
  <zorder>label_2</zorder>
  <zorder>category</zorder>
  <zorder>startScreen</zorder>
  <zorder>title</zorder>
  <zorder>startButton</zorder>
  <zorder>selectSeconds</zorder>
  <zorder>secondsSlider</zorder>
  <zorder>secondText</zorder>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>secondsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>selectSeconds</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
